File: Python_tutorials/19_mini_projects/01_calculator/calculator.py
def calculate(a, operator, b):
    if operator == "+": return a + b
    if operator == "-": return a - b
    if operator == "*": return a * b
    if operator == "/":
        if b == 0: raise ZeroDivisionError("Cannot divide by zero")
        return a / b
    raise ValueError("Unknown operator")

print("Simple Calculator")
try:
    a = float(input("First number: "))
    operator = input("Operator (+ - * /): ").strip()
    b = float(input("Second number: "))
    print("Result:", calculate(a, operator, b))
except (ValueError, ZeroDivisionError) as error:
    print("Error:", error)

File: JavaScript_tutorials/19_intermediate_projects/01_calculator/calculator.js
function calculate(a,op,b){ if(op==='+')return a+b; if(op==='-')return a-b; if(op==='*')return a*b; if(op==='/'){if(b===0)throw new Error('Division by zero');return a/b;} throw new Error('Unknown operator'); }
console.log(calculate(10,'*',5));
